SPEC.md

```diff
@@ -5053,19 +5057,21 @@ workflow name {
 
 ### Workflow Elements
 
-Tasks and workflows have several elements in common. These sections have nearly the same usage in workflows as they do in tasks, so we just link to their earlier descriptions.
+Tasks and workflows have several elements in common. When applicable, the task definition for these sections is linked to rather than duplicated. 
 
-* [`input` section](#task-inputs)
-* [Private declarations](#private-declarations)
-* [`output` section](#task-outputs)
-* [`meta` section](#metadata-sections)
-* [`parameter_meta` section](#parameter-metadata-section)
+A workflow is comprised of the following elements:
 
-In addition to these sections, a workflow may have any of the following elements that are specific to workflows:
+* A singular, optional [`input`](#task-inputs) section (_identical to the `input` section within tasks_).
+* A singular, optional [`output`](#task-outputs) section (_identical to the `output` section within tasks_).
+* A singular, optional [`meta`](#metadata-sections) section (_identical to the `meta` section within tasks_).
+* A singular, optional [`parameter_meta`](#parameter-metadata-section) section (_identical to the `parameter_meta` section within tasks_).
+* Any number of workflow execution elements, which include the following:
+  * A [private declaration](#private-declarations) (_identical to the private declarations within tasks_).
+  * A [`call`](#call-statement) statement, which invokes tasks or subworkflows.
+  * A [`scatter`](#scatter-statement) statement, which enables parallelized of workflow execution elements across collections.
+  * A [conditional (`if`)](#conditional-statement) statement, which enables conditional execution of workflow execution elements.
 
-* [`call`s](#call-statement) to tasks or subworkflows
-* [`scatters`](#scatter), which are used to parallelize operations across collections
-* [Conditional (`if`)](#conditional-if-block) statements, which are only executed when a conditional expression evaluates to `true`
+There is no enforced ordering of workflow elements.
```

Comment:
Explicitly state that there is no ordering of elements within workflows.

Reply:
I think it'd be good to include a bit suggesting people order them in a chronological fashion to improve readability.

Reply:
I have updated the description in [df11c9c](https://github.com/openwdl/wdl/pull/598/commits/df11c9ce09fec576a2bd628286d187bfeb8b7d5a), let me know what you think.